app/views/movies/_movie_partial_loop.html.erb

```diff
@@ -1,24 +1,20 @@
 <div class="row tile-container">
   <% movies.each do |movie| %>
     <article class="tile-cover-pic">
-      <!-- MODAL TRIGGER -->
+      <!-- MODAL TRIGGER 1 of 2 -->
```

Comment:
I tried to make the existence of these 2 different modal triggers more obvious since I missed it the first few times I looked at this file.